cachito/web/static/api_v1.yaml

```diff
@@ -185,6 +185,78 @@ paths:
               $ref: '#/components/schemas/RequestUpdate'
       security:
       - "Kerberos Authentication": []
+  "/requests/{id}/configuration-files":
```

Comment:
Can we add a link to the configuration-files from the request object in the API response? This will make it simpler for clients to realize there's additional information they need to fetch. I'd always include the link, even if there are no configuration files.

Reply:
Did this make into api_v1.yaml?

Reply:
@mprahl ping

Reply:
@lcarva just added it. Thanks!